action.yml

```diff
@@ -66,7 +53,7 @@ runs:
       if: ${{ steps.check_exclude_file.outputs.files_exists == 'false' && steps.check_dockerfile.outputs.files_exists == 'true' }}
       uses: sclorg/testing-farm-as-github-action@v1
       with:
-        api_key:  ${{ steps.api_key.outputs.value }}
+        api_key:  ${{ inputs[steps.vars.outputs.api_key] }}
```

Comment:
This is not needed. The wrapper is used by us. The input variable from containers is beside.

Reply:
Please look at this comment: https://github.com/sclorg/nginx-container/pull/235#issuecomment-1429833400.

Reply:
I don't know if exposing private secrets is a good security practice. Usually, when you need access to repository secrets from the fork you use `on: workflow_run` or `on: pull_request_target` triggers.

https://securitylab.github.com/research/github-actions-preventing-pwn-requests/